Dockerfile_mail_filter

```diff
@@ -0,0 +1,35 @@
+# https://hub.docker.com/_/debian
+FROM docker.io/debian:testing-slim AS build
+MAINTAINER Jelmer Vernooij <jelmer@jelmer.uk>
+
+# Enable forward compatibility with newer versions of Python
+ENV PYO3_USE_ABI3_FORWARD_COMPATIBILITY="1"
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update --yes \
+ && apt-get install --yes --no-install-recommends \
+        auto-apt-proxy \
+        iproute2 \
```

Comment:
why do we need iproute2?

Reply:
Otherwise:

> /usr/bin/auto-apt-proxy: 242: ip: not found

Reply:
Hmm, that looks like a missing dependency in auto-apt-proxy. Can you file a bug?

Reply:
Thank you!